spk/redis/Makefile

```diff
@@ -1,28 +1,35 @@
 SPK_NAME = redis
-SPK_VERS = 3.0.5
-SPK_REV = 5
+SPK_VERS = 5.0.7
+SPK_REV = 6
 SPK_ICON = src/redis.png
+DSM_UI_DIR = app
 
 DEPENDS = cross/busybox cross/$(SPK_NAME)
 
-MAINTAINER = Diaoul
+MAINTAINER = DigitalBox
 DESCRIPTION = Redis is an open source, advanced key-value store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets and sorted sets.
+
 RELOAD_UI = yes
-DISPLAY_NAME = Redis
-CHANGELOG = "Update to 3.0.5"
+STARTABLE = yes
+DISPLAY_NAME = redis
+
+CHANGELOG = "Update to 5.0.7"
+
+# PPC_ARCHES except qoriq are not supported anymore
+UNSUPPORTED_ARCHS = powerpc ppc824x ppc853x ppc854x
 
 HOMEPAGE = http://redis.io
 LICENSE  = 3-clause BSD
 
-INSTALLER_SCRIPT = src/installer.sh
-SSS_SCRIPT       = src/dsm-control.sh
-CONF_DIR         = src/conf/
+SERVICE_USER = auto
+SERVICE_SETUP = src/service-setup.sh
+NO_SERVICE_SHORTCUT = yes
 
 INSTALL_PREFIX = /usr/local/$(SPK_NAME)
 
 POST_STRIP_TARGET = redis_extra_install
 
-BUSYBOX_CONFIG = usrmng
+BUSYBOX_CONFIG = usrmng daemon
 ENV += BUSYBOX_CONFIG="$(BUSYBOX_CONFIG)"
```

Comment:
As far as I understand BusyBox is useless here... These lines and dependency should be removed.

Reply:
@ymartin59 sorry for not validate this. As the redis package already depended on busybox I have overseen this.

@DigitalBox98 can you please verify the need of busybox or remove it?

Reply:
I am almost sure `usrmng` is no longer used and `daemon` has just been introduced.
Previous package dsm-control.sh relies on `PID_FILE` produced by `redis-server`, so it may be possible to control processes with busybox daemon service tool.

Reply:
I've tested by removing the busybox dependency and ymartin59 is right : it can be removed and redis server is working fine.

Unfortunately on my side I can't propose a new PR to remove this, as I've another one pending on the master branch of my repo (postgresql). But let me know if I can help on this.

Reply:
@DigitalBox98 Please learn how to create dedicated private branch for each PR instead of committing on master clone from spksrc, it will really help. https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request
You may also read https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks
By the way, I think you will not be able to submit another PR from your master clone. You should create an dedicated branch from new spksrc master and cherry-pick your commit(s).

Reply:
I usually create a branch before each PR, but it was a mistake for the last one. I will recreate a branch and a fresh new PR for postgresql :)

For redis, if I understand well, I can propose another PR (with the branch:) ) for the remarks on the maintainer and the busy box part ?

Reply:
Yes of course. We do not revert PR as it results of an ugly history.

Reply:
Busybox dependency and maintainer name are now OK:)
A branch has been created for that and new PR #3839 submitted